### PR TITLE
add a FAQ item for how to make the installed tool visible in VSCode Extension

### PR DESCRIPTION
**Pull request by ChenJieting**
# Description

We noticed that some users cannot see the tool in the tool list of VSCode Extension after they installed it via `pip install [tool-package-name]`. They may need to reload the window to refresh. So we add a FAQ item in docs/how-to-guides/faq.md.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [x] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by 16oeahr — APPROVED** [review]

File: docs/how-to-guides/faq.md [hunk 1]
```diff
@@ -33,3 +33,12 @@ For more detail about keyring third-party backend, please refer to 'Third-Party
 If you are using WSL, this is a known issue for `webbrowser` under WSL; see [this issue](https://github.com/python/cpython/issues/89752) for more information. Please try to upgrade your WSL to 22.04 or later, this issue should be resolved.
 
 If you are still facing this issue with WSL 22.04 or later, or you are not even using WSL, please open an issue to us.
+
```
Comment (chjinche) [inline docs/how-to-guides/faq.md:36]: You could use GPT to refine the content and to be more professional. For example:

Here is my attempt to refine the troubleshooting documentation:

## Troubleshooting 

### Installed tool not appearing in VSCode Extension tool list

After installing a tool package with `pip install [tool-package-name]`, the new tool may not immediately appear in the tool list within the VSCode Extension, as shown below:

VSCode Extension tool list

This is often due to outdated cache. To refresh the tool list and make newly installed tools visible:

1. Open the VSCode Extension window.

2. Bring up the command palette by pressing Ctrl+Shift+P.

3. Type and select the "Developer: Reload Webviews" command. 

4. Wait a moment for the tool list to refresh.

Reloading clears the previous cache and populates the tool list with any newly installed tools. After reloading, verify that the missing tools are now visible before trying to use them in the extension.

**Comment by chjinche** [comment]
Please add clear and informative description.

**Comment by ChenJieting** [comment]
> Please add clear and informative description.

Done

File: docs/how-to-guides/faq.md (hunk 1 above)
Comment (ChenJieting) [reply docs/how-to-guides/faq.md:36]: > You could use GPT to refine the content and to be more professional. For example:
> 
> Here is my attempt to refine the troubleshooting documentation:
> 
> ```
> ## Troubleshooting 
> 
> ### Installed tool not appearing in VSCode Extension tool list
> 
> After installing a tool package with `pip install [tool-package-name]`, the new tool may not immediately appear in the tool list within the VSCode Extension, as shown below:
> 
> VSCode Extension tool list
> 
> This is often due to outdated cache. To refresh the tool list and make newly installed tools visible:
> 
> 1. Open the VSCode Extension window.
> 
> 2. Bring up the command palette by pressing Ctrl+Shift+P.
> 
> 3. Type and select the "Developer: Reload Webviews" command. 
> 
> 4. Wait a moment for the tool list to refresh.
> 
> Reloading clears the previous cache and populates the tool list with any newly installed tools. After reloading, verify that the missing tools are now visible before trying to use them in the extension.
> ```

Thanks! That looks better. I have updated the content according to your advice.

**Review by chjinche — APPROVED** [review]

**Review by 16oeahr — APPROVED** [review]

**Pull request merged**